File: python/monotonic_array.py
def monotonic_array(array):
    if len(array)<=1:
        return True
    if array[0]<array[len(array)-1]:
        for i in range(len(array)-1):
            if array[i+1]<array[i]:
                return False
        return True
    else:
        for i in range(len(array)-1):
            if array[i+1]>array[i]:
                return False
        return True
    
print(monotonic_array([9,1,1,3,6]))

File: javascript/monotonic_array.js
const monotonic_array = (arr) => {
    if (arr.length <= 1) {
        return true
    }
    if (arr[0] < arr[arr.length - 1]) {
        for (let i = 0; i < arr.length - 1; i++) {
            if (arr[i + 1] < arr[i]) {
                return false
            }
        }
        return true
    } else {
        for (let i = 0; i < arr.length - 1; i++) {
            if (arr[i + 1] > arr[i]) {
                return false
            }
        }
        return true
    }
}

console.log(monotonic_array([-1, 0, 3, 5]));